src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -127,6 +322,33 @@ public void putCreateExternalBinaryContentProxy(final String uri) {
                         "request entity, as indicated by a link with " +
                         "rel=\"http://fedora.info/definitions/fcrepo#ExternalContent\"",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final String location = uri + info.getId();
```

Comment:
It would be better if you check to see if the base URL does not end with a `/`. If so add in a slash.